### PR TITLE
week2 homework

### PR DESCRIPTION
**Pull request by suvarna-ratna**

### CODE REVIEW

File: week2/class2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+//Caclulator
+app.MapGet("/calc", (string numberOne,string numberTwo,string operation) => calcFun(numberOne,numberTwo,operation));
+string calcFun(string numberOne,string numberTwo,string operation)
```
Comment (calindurnea) [inline week2/class2/Program.cs]: compared to `JS`, in `C#` is much more common to use PascalCase when naming methods, e.g `calcFun => CalcFun` 😊

**Review by calindurnea — APPROVED** [review]

File: week2/class2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+//Caclulator
+app.MapGet("/calc", (string numberOne,string numberTwo,string operation) => calcFun(numberOne,numberTwo,operation));
+string calcFun(string numberOne,string numberTwo,string operation)
+{
+    var numberOneIsValid = int.TryParse(numberOne, out var parsedNumberOne);
+    var numberTwoIsValid = int.TryParse(numberTwo, out var parsedNumberTwo);
+
+    if(!numberOneIsValid || !numberTwoIsValid){
+        return "one of the two numbers is not valid";
```
Comment (calindurnea) [inline week2/class2/Program.cs:14]: You can use `Results.BadRequest()` if you want to also return a 400 status code

**Review by calindurnea — APPROVED** [review]
Good job! 🥇 
I left a few comments, let me know if you have any questions.

To have a more uniform formatting, you can use `Alt+Shift+F` in Visual Studio to auto format the code 😊

File: week2/class2/Program.cs (hunk 1 above)
Comment (suvarna-ratna) [reply week2/class2/Program.cs]: Thank you .Done

File: week2/class2/Program.cs (hunk 2 above)
Comment (suvarna-ratna) [reply week2/class2/Program.cs:14]: okay

**Pull request merged**